disasterinfosite/templates/index.html

```diff
@@ -108,6 +108,16 @@ <h5 class>What to Expect</h5>
 {% include "geek_box.html" %}
 
 <div class="loading hide"></div>
+<!-- Here's the script for Google Analytics. -->
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
some people get even cuter and decided to spell something else! I thought about doing h,a,z,a,r,d,s...